api/fees/fees.go

```diff
@@ -112,7 +112,7 @@ func (f *Fees) handleGetFeesHistory(w http.ResponseWriter, req *http.Request) er
 
 	return utils.WriteJSON(w, &FeesHistory{
 		OldestBlock:   oldestBlockRevision,
-		BaseFees:      baseFees,
+		BaseFeePerGas: baseFees,
```

Comment:
If using singular for this one the same criteria should apply to `GasUsedRatios` below (it should be just `GasUsedRatio`). See `Returns` [here](https://docs.metamask.io/wallet/reference/json-rpc-methods/eth_feehistory/) as a reference.

Reply:
Makes sense, applied that change 👍